### PR TITLE
Update Troubleshooting.md

### PR DESCRIPTION
**Pull request by jesp209i**
add text about removing `updating` marker

## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Cloud / CICD

## Deadline (if relevant)

no deadline

### CODE REVIEW

**Comment by alina-tincas** [comment]
Hi @jesp209i thank you for the PR 🙌 

We will take a look at it as soon as we can 😊

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md [hunk 1]
```diff
@@ -33,6 +33,23 @@ Commit the change to your repository and run the pipeline.
...
+
+1. Access KUDU on the "left-most" environment
+  * If you only have one environment you want the live environemnt
+  * If you have more than one environment, you want the development environment
+
+3. Navigate to `site` > `locks` folder
+
+4. In the folder, there should be a file named `updating`. Remove it
+
+Once the leftover marker file is removed, you are ready to run your pipeline again.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md:51]:
```suggestion
Once the leftover marker file is removed, you run your pipeline again.
```

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md [hunk 2]
```diff
@@ -33,6 +33,23 @@ Commit the change to your repository and run the pipeline.
 
 If you want you can change the filename back to `Readme.md` after a successful CI/CD deployment.
 
+### The site can't be upgraded as it's blocked with the following markers: updating
+
+In rare cases deployments fail, and the cloud infrastructure doesn't clean up correctly. This leaves behind the "updating" marker.
+The next time you try to deploy through your pipeline you encounter the error.
+
+In order to fix this issue, you would need to use [KUDU](../../power-tools/README.md) to remove the leftover marker file.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md:41]:
```suggestion
In order to fix this issue, you need to use [KUDU](../../power-tools/README.md) to remove the leftover marker file.
```

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md [hunk 3]
```diff
@@ -33,6 +33,23 @@ Commit the change to your repository and run the pipeline.
 
 If you want you can change the filename back to `Readme.md` after a successful CI/CD deployment.
 
+### The site can't be upgraded as it's blocked with the following markers: updating
+
+In rare cases deployments fail, and the cloud infrastructure doesn't clean up correctly. This leaves behind the "updating" marker.
+The next time you try to deploy through your pipeline you encounter the error.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md:39]:
```suggestion
The next time you try to deploy through your pipeline you will encounter an error.
```

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md [hunk 4]
```diff
@@ -33,6 +33,23 @@ Commit the change to your repository and run the pipeline.
 
 If you want you can change the filename back to `Readme.md` after a successful CI/CD deployment.
 
+### The site can't be upgraded as it's blocked with the following markers: updating
+
+In rare cases deployments fail, and the cloud infrastructure doesn't clean up correctly. This leaves behind the "updating" marker.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md:38]:
```suggestion
In rare cases deployments fail, and the cloud infrastructure doesn't clean up correctly. This leaves behind an "updating" marker.
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@jesp209i Thanks for the PR!

Looks good, just have a couple of suggestions 😄

**Comment by jesp209i** [comment]
Great suggestions, thanks! - they have all been applied 👍

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md [hunk 5]
```diff
@@ -33,6 +33,23 @@ Commit the change to your repository and run the pipeline.
...
+In order to fix this issue, you need to use [KUDU](../../power-tools/README.md) to remove the leftover marker file.
+
+1. Access KUDU on the "left-most" environment
+  * If you only have one environment you want the live environemnt
+  * If you have more than one environment, you want the development environment
+
+3. Navigate to `site` > `locks` folder
+
+4. In the folder, there should be a file named `updating`. Remove it
+
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md:50]:
```suggestion
1. Access KUDU on the "left-most" environment
  * If you only have one environment you want the live environment
  * If you have more than one environment, you want the development environment

3. Navigate to `site` > `locks` folder  In there, there should be a file named `updating`

4. Remove the `updating` file.

```

**Review by jonat123 — APPROVED** [review]
@jesp209i Thanks for merging in my suggestions! :D 

Just noticed some minor things, and made suggestions for them and added them to speed up the process 😄

I will go ahead and merge in this PR!

**Pull request merged**